### ISSUE TITLE
vidsrc.to server is down, NEW: vidsrc2.to

### ISSUE THREAD

**Issue by Internetezoo** [Prudhvi-pln/udb#23]
Hello!

Can it be changed to **vidsrc.cc**? Thx!

**Comment by Internetezoo** [comment]
Hello!
vidsrc2.to is a new server, can you adapt the API for it?

Original (Ciarands) keys.json modified, now https://github.com/Inside4ndroid/vidkey-js/blob/main/keys.json
it still works.

**Comment by Prudhvi-pln** [comment]
Hi @Internetezoo, Thanks for the details!. Sorry, I've been busy for a while and couldn't spend much time on this. I'm planning to work on it, this weekend.

**Comment by Prudhvi-pln** [comment]
Looks like vidsrc2.to is also down now 😞

**Comment by Prudhvi-pln** [comment]
Hi @francqz31, thanks for the wonderful citings and for using UDB 🙂.
I am keeping an eye on vidsrc, but it has been unstable for the past few months. So, I am hesitant to use vidsrc.
However, vidbinge looks promising. Let me observe if it is stable and give it a try.

**Comment by Prudhvi-pln** [comment]
Fortunately, the anime sites I use in UDB seems to be fine. https://anitaku.pe/home.html and https://animepahe.ru/ are working.
However, the drama site (https://myasiantv.ac/) is affected. I need to look for an alternative to this as well.

**Comment by Internetezoo** [comment]
The **vidsrc.to** page is working again

**Comment by ZetsApocNow** [comment]
hi any luck with fixing the downloaders for movies and tv @Prudhvi-pln ? really do miss this tool

**Comment by Prudhvi-pln** [comment]
Hello! Sorry for the delayed response, I have been busy with other stuff and couldn't get much time to spend on this.
Actually, a lot of sites went down (including Anime, Drama). So, I'm working on restoring them.
Parallelly, I was also checking on the repos shared above, but they have their implementation in typescript and I'm struggling to convert it to Python 😫

**Comment by Prudhvi-pln** [comment]
Hello All, Thanks for waiting. UDB is back with v2.14.0.
This version restores support for Anime, Drama, Movies & TV Series as well. 

Please use it and let me know in case of any issues.

**Comment by Internetezoo** [comment]
https://www.cineby.app ???

**Comment by Prudhvi-pln** [comment]
Yes, but they have a hell lot of encryption and requires active maintenance. Will add it if I find an easy way to scrape it. (maybe using browser automation)

**Comment by Prudhvi-pln** [comment]
Closing this issue, as this requires huge effort to maintain it. However, will track this in my backlog.
